README.md

```diff
@@ -32,12 +32,19 @@ follow embedded-wg best practice
 This group will, eventually, try to maintain:
 
 * PACs:
-  * EFM32GG (working; named efm32gg990)
-  * EFR32xG1 (working)
-  * EFM32HG (working; named efm32hg309f64-pac)
+  * [EFM32GG](https://github.com/maximeborges/svd2rust_efm32gg990)
+  * [EFR32xG1](https://github.com/em32-rs/efr32xg1-pac)
+  * [EFM32HG](https://github.com/em32-rs/efm32hg-pac)
   * EFM32G, EFM32TG (at least there's hardware around to test it)
   * all the others (as soon as someone gets the hardware to test them)
 
-* A single HAL crate: em32-hal (to be merged from efm32gg-hal and imtomu-rs)
+* A single HAL crate: em32-hal (to be merged from [efm32gg-hal](https://github.com/chrysn/efm32gg-hal) and [imtomu-rs](https://github.com/fudanchii/imtomu-rs))
 
 * yet indeterminate Board Support Crates
+
+
+## Related projects
+
+* [Silicon Labs software](http://devtools.silabs.com/dl/documentation/doxygen/5.7/index.html<Paste>):
+  Various C libraries maintained by the chips' manufacturer, including emlib (their hardware abstraction) and board support packages
+* [RustyGecko](https://github.com/RustyGecko): Wrapping emlib via the FFI to Rust. Last updated 2015.
```

Comment:
As opposed to sucking in all bsps, I think an 'awesome em rust' list that has the tomu bsp would be a good header addition here.

Also I think a call to action, and a link to our shared goals ask to join up and support the commons would be good, assuming we would know what to do with the help if we got it.